src/python_bluesky_taskgraph/core/types.py

```diff
@@ -1,5 +1,5 @@
 from dataclasses import dataclass, field
```

Comment:
I recommend renaming this file because `types` is a module in the standard library.
If it's private (i.e. only used within `core`) you could call it `_types.py`.

Reply:
Done (-> type_hints as some needed external to core)